provider/provider.go

```diff
@@ -56,12 +56,14 @@ func DatabricksProvider() *schema.Provider {
 			"databricks_user":                   identity.ResourceUser(),
 			"databricks_service_principal":      identity.ResourceServicePrincipal(),
 
-			"databricks_mws_customer_managed_keys":  mws.ResourceCustomerManagedKey(),
-			"databricks_mws_credentials":            mws.ResourceCredentials(),
-			"databricks_mws_log_delivery":           mws.ResourceLogDelivery(),
-			"databricks_mws_networks":               mws.ResourceNetwork(),
-			"databricks_mws_storage_configurations": mws.ResourceStorageConfiguration(),
-			"databricks_mws_workspaces":             mws.ResourceWorkspace(),
+			"databricks_mws_customer_managed_keys":   mws.ResourceCustomerManagedKey(),
+			"databricks_mws_credentials":             mws.ResourceCredentials(),
+			"databricks_mws_log_delivery":            mws.ResourceLogDelivery(),
+			"databricks_mws_networks":                mws.ResourceNetwork(),
+			"databricks_mws_storage_configurations":  mws.ResourceStorageConfiguration(),
+			"databricks_mws_workspaces":              mws.ResourceWorkspace(),
+			"databricks_mws_vpc_endpoint":            mws.ResourceVPCEndpoint(),
```

Comment:
could you make those 6 items in alphabetical order? :)

Reply:
done